core/mondoo-azure-inventory.mql.yaml

```diff
@@ -215,4 +275,10 @@ packs:
         docs:
           desc: |
             This query retrieves data for all resources inside the subscription
-        mql: azure.subscription.resources
\ No newline at end of file
+        mql: azure.subscription.resources
+      - uid: mondoo-asset-inventory-azure-mfa-users
```

Comment:
We need to rework this in a data gathering way, not a security way. We could make it "Retrieve all users in the subscription and their MFA status"

Reply:
done, Thanks!